src/manager.rs

```diff
@@ -113,49 +114,49 @@ impl Manager {
         })
```

Comment:
I didn't change the methods above because:
* add/remove_watcher manipulates internal state
* add/remove_driver do not seem threadsafe in the openzwave C++ library